extensions/amp-social-share/1.0/social-share.js

```diff
@@ -70,48 +83,52 @@ export function SocialShare(props) {
  *   baseEndpoint: ?string,
  *   checkedWidth: ?number,
  *   checkedHeight: ?number,
+ *   params: !JsonObject,
  * }}
  */
 function checkProps(props) {
   const {
     'type': type,
-    'shareEndpoint': shareEndpoint,
-    'params': params,
-    'bindings': bindings,
+    'endpoint': endpoint,
+    'target': target,
     'width': width,
     'height': height,
+    'additionalParams': additionalParams,
   } = props;
 
   // Verify type is provided
   if (type === undefined) {
     throw new Error(`The type attribute is required. ${NAME}`);
   }
 
-  // bindings and params props must be objects
-  if (params && !isObject(params)) {
-    throw new Error(`The params property should be an object. ${NAME}`);
-  }
-  if (bindings && !isObject(bindings)) {
-    throw new Error(`The bindings property should be an object. ${NAME}`);
-  }
-
-  // User must provide shareEndpoint if they choose a type that is not
+  // User must provide endpoint if they choose a type that is not
   // pre-configured
   const typeConfig = getSocialConfig(type) || dict();
-  const baseEndpoint = typeConfig['shareEndpoint'] || shareEndpoint;
+  const baseEndpoint = endpoint || typeConfig['shareEndpointPreact'](props);
   if (baseEndpoint === undefined) {
     throw new Error(
-      `A shareEndpoint is required if not using a pre-configured type. ${NAME}`
+      `An endpoint is required if not using a pre-configured type. ${NAME}`
     );
   }
 
+  /*
+  // Handle receipient for type = 'email' without custom endpoint
```

Comment:
Is this section supposed to be commented out?

Reply:
removed, yes it's handled directly within the config file now